packages/gatsby/src/bootstrap/requires-writer.ts

```diff
@@ -195,7 +195,7 @@ export const writeAll = async (state: IGatsbyState): Promise<boolean> => {
   let syncRequires = `${hotImport}
 
 // prefer default export if available
-const preferDefault = m => m && m.default || m
+const preferDefault = m => (m && m.default) || m
```

Comment:
I guess in the future this will be

```suggestion
const preferDefault = m => m?.default ?? m
```

Reply:
In this case I'm not sure which one is more readable :) 

But I think we could use your suggestions because this file _should_ be handled by babel so both optional chaining and nullish coalescing operator should work just fine (tho users do have option to use their own custom babel config, so probably keeping old style would result in less potential for problems)

Reply:
+ it's less bytes